infra/dns_firewall.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_route53_resolver_firewall_domain_list" "allow_domain_list_dataexfiltration" {
+  name    = "allow-list-${local.project}"
+  domains = ["pagopa.it.", "*.pagopa.it.", "io.pagopa.it.", "amazonaws.com.", "*.amazonaws.com.", "*.compute.internal."]
```

Comment:
I think we can remove *.pagopa.it because contains io.pagopa.it
*.compute.internal. also we can remove (by default we do not call vm inside vpc)